### DISCUSSION TITLE
I use UC . The initial JavaScript message on the website causes the driver to freeze.

### DISCUSSION THREAD

**Discussion by gihon12312340** [seleniumbase/SeleniumBase#2353] [Q&A] [answered]
```
from seleniumbase import Driver
from seleniumbase import page_actions
from selenium.webdriver.common.keys import Keys


driver = Driver(uc=True)


driver.get('https://www.recreationcenter01.umc.com/um01.aspx?module=net_booking&files=booking_place&PT=1')


page_actions.send_keys(driver, selector='body', text=(Keys.CONTROL + 't'))
```

Driver freezes upon opening the webpage. How can I resolve this?

**Comment by mdmintz** [comment, marked as answer]
Looks like a page with a Turnstile that must be clicked. See the example for that here: [SeleniumBase/examples/raw_turnstile.py](https://github.com/seleniumbase/SeleniumBase/blob/master/examples/raw_turnstile.py)

`page_actions` should not be used directly. See [Syntax Formats](https://github.com/seleniumbase/SeleniumBase/blob/master/help_docs/syntax_formats.md) for valid ways of structuring tests.

And looks like that page has some Javascript that never finishes loading until the pop-up is closed. Not sure if there's an easy way around it.

**Reply by gihon12312340** [reply]
Can you help me with any method to possibly resolve this JavaScript? I really need to shut it down.

The method I'm currently trying is manually opening a new tab, where the JavaScript disappears when I switch to it. However, my driver freezes. Is there any way to open the webpage in the background without immediately switching to it and then navigate to it later?

**Reply by mdmintz** [reply]
Maybe something with host_resolver_rules - https://github.com/seleniumbase/SeleniumBase/issues/2281
